tools/format-source.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -eux
+. "$(dirname "$0")/env.sh"
+
+goimports -w internal cmd
+gofumpt -l -w internal cmd
```

Comment:
I wonder what's the support for this formatter in editors/IDEs. It's subpar compared to `gofmt`, this might affect developer experience negatively.